web-wallet/src/lib/components/Balance/Balance.svelte

```diff
@@ -47,12 +47,10 @@
       data-tooltip-place="right"
     />
   </p>
-  {#if fiatPrice}
-    <p class="dusk-balance__fiat">
-      <strong>
-        ({fiatFormatter(fiatPrice * tokens)})
-      </strong>
-    </p>
-  {/if}
+  <p class="dusk-balance__fiat">
```

Comment:
I have changed it, although I don't like this approach because the value will first be hidden, then will change to 0,00 and then to the actual value, once the balance has loaded, meaning the component will change its state 3 times until it shows the final result, which seems a bit excessive.

Reply:
Then leave it hidden if the balance is 0 as well: no point in showing it anyway.

Reply:
Yeah, I think this is better.

Reply:
And please add a test for both conditions (balance 0 and fiat undefined)

Reply:
Fiat `undefined` was there, added one for balance 0. [Reference](https://github.com/dusk-network/rusk/pull/1515/files#diff-d0f2549fea658200138f53bc020250ff8d1f4b13323f6bcd519bb7ab60b08a23R73-R80)